command_builder.test.ts

```diff
@@ -9,4 +9,8 @@ Deno.test("Quick example", async () => {
   for await (const line of stream) {
     assertEquals(line, "bug : abcdef");
   }
+
+  const result = await $`echo hello`
+    .xargs((input) => $`echo ${input} world`);
```

Comment:
```suggestion
    .xargs((input) => $`echo ${input} world`.stdout("piped"));
```
Sometimes users do not want to pipe stdout, so I want to specify stdout("piped") in XargsFunction instead of here.